subiquity/server/controllers/drivers.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import asyncio
+import logging
+import subprocess
+from typing import Optional
+
+from subiquitycore.context import with_context
+
+from subiquity.common.apidef import API
+from subiquity.server.controller import SubiquityController
+from subiquity.server.curtin import run_curtin_command
+from subiquity.server.types import InstallerChannels
+
+log = logging.getLogger('subiquity.server.controllers.drivers')
+
+
+class DriversController(SubiquityController):
+
+    endpoint = API.drivers
+
+    autoinstall_key = model_name = "drivers"
+    autoinstall_schema = {
+        'type': 'boolean',
```

Comment:
I'm imaging a world where we want some flexibility here, like we want wifi things but not gpu ones.    If that arises, what would we need to change in autoinstall to adapt to that scenario?

Reply:
Well, I guess we'd need ubuntu-drivers to support that concept to start with... I don't think it does now.